managed-databases/postgresql-and-mysql/how-to/apply-maintenance.mdx

```diff
@@ -35,13 +35,13 @@ These maintenance operations are set up, run, and scheduled by Scaleway, but you
     </Message>
 
 
-4. Click **Apply maintenance now** to apply the maintenance immediately. You are redirected to a screen that shows you the status of the maintenance.
+4. Click **Apply maintenance now** to apply the maintenance immediately. You are redirected to a screen that shows you the status of the maintenance. If you want a more detailed maintenance status, you can [use the API](https://www.scaleway.com/en/developers/api/managed-database-postgre-mysql/#path-database-instances-apply-database-instance-maintenance).
     <Message type="important">
       Your Database Instance will be unavailable during the maintenance. This might take a few minutes.
     </Message>
 
     <Message type="note">
-      If you do not apply the maintenance up to 24 hours before it is set to take place, it will be applied automatically on the scheduled date.
+      If you do not apply the maintenance before the scheduled date, it will be applied automatically on the scheduled date. The scheduled maintenance can take up to 4 hours to be applied starting from the scheduled time.
```

Comment:
MAybe remove one `scheduled` here
```suggestion
      If you do not apply the maintenance before the scheduled date, it will be applied automatically on the scheduled date. The maintenance can take up to 4 hours to be applied starting from the scheduled time.
```

Reply:
there are two types of maintenance, so I deliberately kept the scheduled to make sure the users understand that this applies only to this type.